utilities/transactions/transaction_base.h

```diff
@@ -250,6 +250,8 @@ class TransactionBaseImpl : public Transaction {
 
   WriteBatch* GetCommitTimeWriteBatch() override;
 
+  LockTracker& GetTrackedLocks() { return *tracked_locks_; }
```

Comment:
can we just expose it as `const`? `tracked_locks_` should only be updated internally by the transaction? Also, can we comment on the thread-safety of the returned `LockTracker`?